opencga-app/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>uk.ac.ebi.eva</groupId>
         <artifactId>opencga</artifactId>
-        <version>0.5.6</version>
+        <version>0.5.6-mongo4</version>
```

Comment:
Is it advisable to amend the version in that way ?
I guess 0.6 exists elsewhere but in the unlikely event that we need to change it further I would prefer that we version properly. Maybe we can mark these version with `eva` as prefix or suffix

Reply:
Currently this version is deployable only under the EVA organization as shown in `<groupId>uk.ac.ebi.eva</groupId>`